test/integration/BUILD

```diff
@@ -312,6 +312,30 @@ envoy_cc_test(
     ],
 )
 
+envoy_cc_test(
+    name = "http2_flood_integration_test",
+    srcs = [
+        "http2_flood_integration_test.cc",
+    ],
+    shard_count = 4,
```

Comment:
shard_counts still needed?

Reply:
Yes, since we will be adding more tests for upstream flooding.